src/_includes/macros/filters.html

```diff
@@ -230,6 +230,9 @@
                     {{ _filter_checkbox('calendar', 'Richard Cordray')|safe }}
                 </label>
                 <label class="form-group_item">
+                    {{ _filter_checkbox('calendar', 'Meredith Fuchs')|safe }}
```

Comment:
It hasn't been standardized yet (relevant https://github.com/cfpb/front-end/issues/75). It seems like the trend is toward a space? I've been adding it for new code, but in this case `filters.html` has so many pipes throughout the file that it seemed out of scope. Should I update them here?